OrbitCore/SymbolHelper.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <string>
+
+#include "OrbitModule.h"
+#include "Pdb.h"
+
+struct SymbolHelper {
+ public:
+  static std::vector<std::string> collector_symbol_locations;
+  static std::vector<std::string> visualizer_symbol_locations;
+
+  static bool LoadSymbolsIncludedInBinary(std::shared_ptr<Module>);
+  static bool LoadSymbolsCollector(std::shared_ptr<Module>);
+  static bool LoadSymbolsVisualizer(std::shared_ptr<Module>);
+  static void LoadSymbolsFromDebugInfo(std::shared_ptr<Module>,
+                                       ModuleDebugInfo&);
+  static void FillDebugInfoFromModule(std::shared_ptr<Module>,
+                                      ModuleDebugInfo&);
+  static std::vector<std::string> GetCollectorSymbolLocations() {
+    return SymbolHelper::collector_symbol_locations;
+  }
+  static std::vector<std::string> GetVisualizerSymbolLocations();
+};
```

Comment:
do they have to be static?

Reply:
Looks to me the struct should be a namespace and the std::vectors should only be defined in the cpp-file in a anonymous namespace.

Reply:
The main reason why it is done in this way, is so the tests can override `collector_symbol_locations` and `visualizer_symbol_locations` for "mock locations" aka the locations of the test data. 

I am very open for suggestions on how to do it the cleanest way.

Reply:
One way is to remove static - change it to class, hide vectors. This way it can be used by tests and App.

Reply:
Yes in this case I would also prefer to have a simple class. Otherwise it's just another way to hide a global variable.

Reply:
done